modules/svg/image_loader_svg.cpp

```diff
@@ -35,6 +35,11 @@
 
 #include <thorvg.h>
 
+#ifdef TOOLS_ENABLED
+#include "editor/editor_settings.h"
+#include "editor/editor_themes.h"
+#endif
```

Comment:
There shouldn't be editor dependencies in non-editor code. So it would be better to find a way to pass the editor color map from editor code itself.

Reply:
I'm not aware of a good way to do it. Specific loaders are not globally accessible. Maybe some static interface can be made for this specifically, if that's satisfactory.

Reply:
Yeah I'm not sure either. We can poke reduz's brain and see if we allow ourselves this exception or if we want to improve the interface. A way to register editor-specific remaps via a static interface might do the trick indeed.